.github/workflows/ci.yml

```diff
@@ -15,6 +15,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Don't understand this change as workflow is also used on master

Reply:
do not interfere when triggerred from push event. when triggered from PR, it was using the target ref ( ie master ).